src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/NotifyAttentionNeededLicence.kt

```diff
@@ -0,0 +1,32 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model
+
+import com.fasterxml.jackson.annotation.JsonFormat
+import io.swagger.v3.oas.annotations.media.Schema
+import java.time.LocalDate
+
+@Schema(description = "Describes a licence that needs attention for support")
+data class NotifyAttentionNeededLicence(
```

Comment:
This isn't an actual api representation so might be better to move into the service package - it doesn't need swagger annotations